python/lsst/afw/image/exposure/_multiband.py

```diff
@@ -159,26 +205,19 @@ def computePsfKernelImage(self, position=None):
         Parameters
         ----------
         position: `Point2D` or `tuple`
-            Coordinates to evaluate the PSF. If `position` is `None`
-            then `Psf.getAveragePosition()` is used.
+            Coordinates to evaluate the PSF.
 
         Returns
         -------
         self._psfImage: array
             The multiband PSF image.
         """
-        psfs = []
-        # Make the coordinates into a Point2D (if necessary)
-        if not isinstance(position, Point2D) and position is not None:
-            position = Point2D(position[0], position[1])
-        for single in self.singles:
-            if position is None:
-                psfs.append(single.getPsf().computeKernelImage().array)
-            else:
-                psfs.append(single.getPsf().computeKernelImage(position).array)
-        psfs = np.array(psfs)
-        psfImage = MultibandImage(self.filters, array=psfs)
-        return psfImage
+        return computePsfImage(
+            psfModels=self.getPsfs(),
+            position=position,
+            filters=self.filters,
```

Comment:
Shouldn't all of the references to `filters` in this file be `bands` now?

Reply:
I created DM-36099 to change all occurrences of `filters` to `bands` in the `afw` `multiband` modules.